R/spongebobsay.R

```diff
@@ -38,6 +38,20 @@
     return(words[which.max(nchar(words))])
 }
 
+# Takes a vector of characters and pads them with spaces to the same length
+.pad_strings_to_fixed_length <- function(txt, length) {
```

Comment:
I don't like the idea of using `length` as the name of the argument due to collision with the base R `length` function. Can you use something like `lineLength` instead?

Reply:
I agree. Just pushed a new commit.